ert3/storage/_storage.py

```diff
@@ -1,109 +1,84 @@
 import codecs
-import io
 import json
-import os
 from pathlib import Path
-from typing import Any, Iterable, Optional, Union, cast
+from typing import Any, Iterable, Optional, Union, cast, Mapping
+import io
 
 import cloudpickle
 import requests
-import yaml
 import ert3
 
-_STORAGE_FILE = "storage.yaml"
-_STORAGE_URL = "http://localhost:8000"
-
 
+_STORAGE_URL = "http://localhost:8000"
 _DATA = "__data__"
 _PARAMETERS = "__parameters__"
-
 _ENSEMBLE_RECORDS = "__ensemble_records__"
-
 _SPECIAL_KEYS = (_ENSEMBLE_RECORDS,)
 
 
-def _generate_storage_location(workspace: Union[str, Path]) -> Path:
-    workspace = Path(workspace)
-    return workspace / ert3._WORKSPACE_DATA_ROOT / _STORAGE_FILE
-
-
-def _assert_storage_initialized(storage_location: Path) -> None:
-    if not os.path.isfile(storage_location):
-        raise ValueError("Storage is not initialized")
+def _get_experiment_by_name(experiment_name: str) -> Mapping[str, Any]:
+    response = requests.get(url=f"{_STORAGE_URL}/experiments")
+    assert response.status_code == 200
```

Comment:
Debateable, but asserts can be disabled by the user python-wide so don't use them in production code.